web/index.html

```diff
@@ -2,10 +2,22 @@
 <html lang="en">
   <head>
     <title>My CV v2 - starter</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-    <!-- Change this file -->
-    <h1>Hi! How are you today?</h1>
-    <p>I am the file <code>index.html</code> and you want to change me!</p>
+    <div class="despre-mine">
+      <h1>Gratiela Zburlea</h1>
+      <h2>Junior Web Developer</h2>
+      <h3>About me</h3>
+      <div class="cutie-cu-umbra">
```

Comment:
It's impressive to add custom name classes instead of copying them from the tutorial!
Keep it up!

Reply:
I like this class name :D